features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -195,7 +195,11 @@ export function SidebarNavigation() {
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              onClick={() =>
```

Comment:
As discussed on discord I think using the MenuItemLink component would be easier.